common/common.h

```diff
@@ -437,6 +437,11 @@ std::vector<std::string> string_split<std::string>(const std::string & input, ch
     return parts;
 }
 
+static bool starts_with(const std::string & str,
```

Comment:
If this function is made public, at least it should have a `string_` prefix like the rest.